tools/metricsdocs/metricsdocs.go

```diff
@@ -124,66 +153,100 @@ func createFile() *os.File {
 	return file
 }
 
-func (r *releaseData) writeHeader() {
-	r.outFile.WriteString("# KubeVirt components metrics\n\n")
-	r.outFile.WriteString("This document aims to help users that are not familiar with metrics exposed by all the KubeVirt components.\n")
-	r.outFile.WriteString("All metrics documented here are auto-generated in each component repository and gathered here.\n")
-	r.outFile.WriteString("They reflect and describe exactly what is being exposed.\n\n")
-}
-
-func (r *releaseData) parseMetrics() map[string]string {
-	metrics := make(map[string]string)
+func (r *releaseData) parseMetrics() []metric {
+	var allMetrics []metric
 
 	for _, p := range r.projects {
-		content, err := readLines(path.Join(p.repoDir, "/", p.metricsDocPath))
+		content, err := readLines(filepath.Join(p.repoDir, p.metricsDocPath))
```

Comment:
nit: why changed from path to filepath? 
in general we should keep the pr scope to the changes relevant to the pr subject